biometricutils/build.gradle

```diff
@@ -30,6 +30,9 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    lintOptions {
```

Comment:
What is the reason of disabling linter?

Reply:
as far as I remember, this module failing the linter step on Bitrise, so I disabled it but I forgot to fix it 🤦.